pseudocode.py

```diff
@@ -0,0 +1,126 @@
+Outputs:        None
+Returns:        Boolean
+Assumptions:    N/A
+'''
+def accept_device(exifData, acceptedMobileDevices):
+    for device in range(len(acceptedMobileDevices)):
+        if (exifData[DEVICE] == acceptedMobileDevices[device]):
+            return true
+    return false
+
+'''
+Purpose:        The purpose of this function is to determine whether or not the image was
+                altered from its original form. I.e. do the modification and creation dates coincide.
+Inputs:         dict exifData
+Outputs:        None
+Returns:        Boolean
+Assumptions:    N/A
+'''
+def not_edited(exifData):
+    if (exifData[Create Date] == exifData[Modify Date]):
+        return true
+    else:
+        return false
+
+'''
+Purpose:        The purpose of this function is to determine whether or not the image
+                contains a direct landscape with sky and view.
+Inputs:         dict exifData, ?
+Outputs:        None
+Returns:        Boolean
+Assumptions:    N/A
+'''
+def is_landscape(exifData, ):
+    #UNKNOWN METHOD
+
+'''
+Purpose:        The purpose of this function is to determine whether or not the size of
+                the image is less than or equal to 200kb.
+Inputs:         dict exifData, int imgMaxSize
+Outputs:        None
+Returns:        Boolean
+Assumptions:    N/A
+'''
+def meets_size_limit(exifData, imgMaxSize):
+    if (exifData[File Size] <= imgMaxSize): #parsing may be necessary here "200 kB"
+        return true
+    else:
+        return false
+
+'''
+Purpose:        The purpose of this function  is to determine whether or not the image is
+                an accepted file type.
+Inputs:         dict exifData, list acceptedFileTypes
+Outputs:        None
+Returns:        Boolean
+Assumptions:    N/A
+'''
+def accepted_file_type(exifData, acceptedFileTypes):
+    for fileType in range(len(acceptedFileTypes)):
+        if (exifData[File Type] == acceptedMobileDevices[fileType]):
+            return true
+    else:
+        return false
+
+'''
+Purpose:        The purpose of this function is to determine whether or not the image
+                exceeds the minimum resolution.
+Inputs:         dict exifData, int imgWidthMin, int imgHeightMin
+Outputs:        None
+Returns:        Boolean
+Assumptions:    N/A
+'''
+def meets_resolution_limit(exifData, imgWidthMin, imgHeightMin):
+    if (exifData[Exif Image Width] >= imgWidthMin  && exifData[Exif Image Height] >= imgHeightMin):
+        return true
+    else:
+        return false
+
+'''
+Purpose:        The purpose of this function is to determine whether or not the image was
+                taken by a device with location services enabled for the camera.
+Inputs:         dict exifData
+Outputs:        None
+Returns:        Boolean
+Assumptions:    N/A
+'''
+def location_services_on(exifData):
+    #may need to check that keys Latitude/Longitude exist before calling them below
+    if (exifData[Latitude] != null && exif_data[Longitude] != null):
```

Comment:
This function definition needs working on. If there is no exifData['Latitude'] or exifData['Longitude'] this if statement will throw an error, and there is no means here to handle that exception. Also, again, please expound on the comment "may need to check that keys Latitude/Longitude exist before calling them below" -  how will you check? Take a peek at this tutorial on [Python dictionaries](https://www.tutorialspoint.com/python/python_dictionary.htm). You may want to focus on a way to check if a dictionary has in a key (where 'Latitude' and 'Longitude' are the keys). You should also consult with the exifData sample output to ensure these are the correct keys.